modules/networkelb-dashboard/main.tf

```diff
@@ -0,0 +1,265 @@
+
+      metric              = "aws.networkelb.active_flow_count_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+    query {
+      query_name = "b"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.networkelb.new_flow_count_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+  }
+
+  chart {
+    name = "New Flow Count _ TLS"
+    rank = "2"
+    type = "timeseries"
+
+    query {
+      query_name = "d"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.networkelb.new_flow_count___tls_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+    query {
+      query_name = "c"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.networkelb.active_flow_count___tls_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+  }
+
+  chart {
+    name = "Processed Bytes"
+    rank = "3"
+    type = "timeseries"
+
+    query {
+      query_name = "a"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.networkelb.processed_bytes_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+    query {
+      query_name = "b"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.networkelb.consumed_lc_us_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+  }
+
+  chart {
+    name = "Client TLS Negotiation Error Count"
+    rank = "4"
+    type = "timeseries"
+
+    query {
+      query_name = "d"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.networkelb.client_tls_negotiation_error_count_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+    query {
+      query_name = "c"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.networkelb.target_tls_negotiation_error_count_count"
+      timeseries_operator = "delta"
+
+
+      group_by {
+        aggregation_method = "sum"
+        keys               = []
+      }
+
+    }
+
+  }
+
+  chart {
+    name = "TCP _ Client _ Reset _ Count"
+    rank = "5"
+    type = "timeseries"
+
+    query {
+      query_name = "d"
+      display    = "bar"
+      hidden     = false
+
+      metric              = "aws.networkelb.tcp___client___reset___count_count"
```

Comment:
These metric names follow a convention I haven't seen before. Let's call it "dunder snake." Can you look into where the determination exists in process or code that this is how we translate these metrics? I think that would help us pinpoint the place to make some high impact changes. Also if you could document why count is appended here it would be great.

Reply:
This might a one of those weird edge case related to Grafana metric name conversion. Will work on this one. God catch.